testsuite/testsuite-management/src/test/resources/secure-management-properties.yml

```diff
@@ -0,0 +1,19 @@
+swarm:
+  management:
+    bind:
+      interface: management
+    http-interface-management-interface:
+      console-enabled: false
+      http-upgrade:
+        enabled: true
+      security-realm: ManagementRealm
+      socket-binding: management-http
+    security-realms:
+      ManagementRealm:
+        plug-in-authentication:
```

Comment:
Should be able to use `in-memory-authentication`, as in https://github.com/rhoar-qe/thorntail-test-suite/blob/master/javaee/ejb-remote-security/src/main/resources/project-defaults.yml#L7-L12